gno.land/pkg/gnoweb/markdown/golden/ext_columns/invalid_beyond_maximum_level_heading.golden.txtar

```diff
@@ -0,0 +1,26 @@
+-- input.md --
+<gno-columns>
+####### Title 1
+content 1
+## Title 2
+content 2
+####### Title 3
+content 3
+## Title 4
+content 4
+</gno-columns>
+
+-- output.html --
+<!-- gno-columns error: invalid columns format: open tag should be followed by heading separator or a closing tag -->
+<!-- unexpected/invalid "ColumnTagOpen" omitted -->
+<p>####### Title 1
+content 1</p>
+<!-- unexpected/invalid "ColumnTagSep" omitted -->
```

Comment:
IMO these two `invalid "ColumnTagSep"` errors (this one and the one bellow) make no sense / can be misleading for the user since there is no separator tag, just regular headings. I would only display this kind of error to the user for invalid `<gno-columns>` / `</gno-columns>` tags.

And even on the parser side, I don't understand why your parser continue to consider heading as separator since there is no `<gno-columns>` currently opened?

Reply:
Fixed in your last commit https://github.com/gnolang/gno/pull/3763/commits/fada00cfa5dc4763bfe73500e3fc7ca544dafbed 👍